docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express')
+const { HTTPReceiver } = require('cloudevents-sdk')
+const { CloudEvent, StructuredHTTPEmitter } = require('cloudevents-sdk/v1')
+const Constants = require('cloudevents-sdk/lib/bindings/http/constants.js')
+
+const target = process.env.K_SINK
+const app = express()
+const receiver = new HTTPReceiver()
+
+const main = () => {
+  app.listen(8080, function () {
```

Comment:
Can we make this `$PORT`?  I wonder if I messed that up in the Go one 😬

Reply:
I had a variable, but then remove since the Go didn't have one. I will add